pkg/controller/clusterprovision/clusterprovision_controller.go

```diff
@@ -272,6 +276,20 @@ func (r *ReconcileClusterProvision) reconcileRunningJob(instance *hivev1.Cluster
 
 	pLog = pLog.WithField("job", job.Name)
 
+	installPod, err := r.getInstallPod(job, pLog)
+	if err != nil {
+		pLog.WithError(err).Error("could not get install pod")
+		return r.transitionStage(instance, hivev1.ClusterProvisionStageFailed, "InstallPodMissing", err.Error(), pLog)
```

Comment:
I am guessing that you are expecting that this will not be hit until the 60 seconds has elapsed, which is not guaranteed.